Makefile

```diff
@@ -23,6 +23,8 @@ up_with_grafana: ## Starts the environment with Grafana
 down: ## Stops the environment
 	docker-compose -f ${dc_path} down
 	env UID=${UID} docker-compose -f ${dc_grafana_path} down
+	rm -rf _dev/clickhouse
```

Comment:
@ivan-stankov-golang This deletes all the ClickHouse persisted beacon data. What's the motivation behind deleting the folder?

Reply:
I created a new target down/clean. To be able to test migrations scripts applied on clean database